src/main/java/io/specto/hoverfly/junit/api/HoverflyOkHttpClientFactory.java

```diff
@@ -0,0 +1,75 @@
+package io.specto.hoverfly.junit.api;
+
+import io.specto.hoverfly.junit.core.HoverflyConstants;
+import io.specto.hoverfly.junit.core.config.HoverflyConfiguration;
+
+public class HoverflyOkHttpClientFactory implements HoverflyClientFactory {
+
+    public HoverflyClient createHoverflyClient(HoverflyConfiguration hoverflyConfig) {
+        return custom()
+                .scheme(hoverflyConfig.getScheme())
+                .host(hoverflyConfig.getHost())
+                .port(hoverflyConfig.getAdminPort())
+                .withAuthToken()
```

Comment:
`withAuthToken()` is optional. It's better to expose the builder for the user to customize the client: 
```
HoverflyClient.custom().withConfig(configuration).withAuthToken();
```